src/services/poller.ts

```diff
@@ -35,7 +35,7 @@ export class Poller {
     // private id = Symbol(Math.floor(Math.random() * 100))
     private polls: ActivePoll[] = []
     constructor(options: IPollerOptions) {
-        global.setInterval(async () => await this.poll(), options.interval)
```

Comment:
I can't remember, but did this `await` get added to fix one the `undefined` errors? I would check and see before removing

Reply:
Looks like the await has always been there : https://github.com/microsoft/ConversationLearner-UI/pull/619/files#diff-079185a02aae96b096f8166810b1a099R35